src/main/flight/pid.h

```diff
@@ -75,6 +75,7 @@ typedef struct pidProfile_s {
 
     uint16_t yaw_lpf_hz;                    // Additional yaw filter when yaw axis too noisy
     uint16_t dterm_lpf_hz;                  // Delta Filter in hz
+    uint16_t dterm_lpf_2_hz;                // Extra PT1 Filter on D in hz
```

Comment:
I'd been standardizing names and suggest the following:

`dterm_lowpass_hz`
`dterm_lowpass2_hz`

Reply:
yes no problem

Reply:
Actually no, decided to keep 'lpf' for the units themselves (the actual values), in CLI they are all 'lpf', so I will keep 'dterm_lpf_hz' etc here.  I think perhaps it's easier to read that way, 'lpf' is a value, 'lowpass' is a filter.

Reply:
I suggested renaming it in CLI as well to make it uniform with `gyro_lowpass_hz`